api/api.py

```diff
@@ -1,9 +1,8 @@
 import connexion
+import os
```

Comment:
No need for `os` anymore if we do the `if __name__ == "__main__"` instead.

Reply:
fixed!